arch/arm/core/aarch32/cortex_m/fpu.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019,2020 Linaro Limited
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <arch/arm/aarch32/cortex_m/cmsis.h>
+
+/**
+ * @file @brief Helper functions for save and restoring the FP context
+ *
+ */
+
+int save_fp_context(uint32_t *buffer, size_t buffer_size_words, bool *context_saved)
+{
+#if defined(CONFIG_FPU_SHARING)
+	if (buffer_size_words < 33) {
```

Comment:
Could the 33 be a macro, for ease of understanding, or, alternatively, add a comment?